plasmapy/particles/particle_class.py

```diff
@@ -466,6 +472,41 @@ def __str__(self) -> str:
         """Return the particle's symbol."""
         return self.particle
 
+    def to_dict(self) -> dict:
+        """
+        Return a dictionary representation of the minimum description of a particle.
+
+        Examples
+        --------
+        >>> lead = Particle('lead')
+        >>> lead.to_dict()
+        {'type': 'Particle', 'symbol': 'Pb'}
+        >>> electron = Particle('e-')
+        >>> electron.to_dict()
+        {'type': 'Particle', 'symbol': 'e-'}
+
+        """
+        particle_dict = dict()
+        particle_dict['type'] = 'Particle'
+        particle_dict['symbol'] = self.particle
+        return particle_dict
+
+    def to_json(self) -> str:
+        """
+        Return a JSON string representation of the minimum description of a particle.
+
+        Examples
+        --------
+        >>> lead = Particle('lead')
+        >>> lead.to_json()
+        '{"type": "Particle", "symbol": "Pb"}'
+        >>> electron = Particle('e-')
+        >>> electron.to_json()
+        '{"type": "Particle", "symbol": "e-"}'
+
+        """
+        return super().to_json(self.to_dict())
```

Comment:
Then (see above) you wouldn't need to implement this `to_json` method in each subclass - you just create `to_dict`.

Reply:
Yeah, that definitely helps. It will also improve testing since we don't need to add additional (and unnecessary) code in the class. Thanks for the tip :)